portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -55,11 +65,15 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
       String name = (String) entity.getProperty("name");
       String comment = (String) entity.getProperty("comment");
+      String image = (String) entity.getProperty("image");
+
+      String content = comment + " " + image;
```

Comment:
At this point I feel that a comment class should be added, it feels like you're doing a hack so that a single string can carry all information. Let's create a Comment class with user, comment, and image, it's cleaner this way and also when you want to add more in the future it would be easier.